static/dist/global-nav.js

```diff
@@ -0,0 +1 @@
.5rem;top:1.5rem;width:.8175rem}.global-nav .global-nav__expanding-row.is-active:after{-webkit-transform:rotate(180deg);transform:rotate(180deg)}.global-nav .global-nav__expanding-row.is-active+.global-nav__split-list{display:block}.global-nav .global-nav__matrix{display:flex;flex-wrap:wrap;list-style:none;margin:0;padding:0}@media (max-width:875px){.global-nav .global-nav__matrix{flex-direction:column}}.global-nav .global-nav__matrix-item{border-top:1px solid #666;flex:1 1 auto;flex-wrap:wrap;margin:0;padding:1rem;width:33.33%}.global-nav .global-nav__matrix-item:nth-child(3n+1){padding-left:0}.global-nav .global-nav__matrix-item:nth-child(3n+3){border-right:0;padding-right:0}@media (min-width:875px){.global-nav .global-nav__matrix-item:first-child,.global-nav .global-nav__matrix-item:nth-child(2),.global-nav .global-nav__matrix-item:nth-child(3){border-top:0}}@media (max-width:875px){.global-nav .global-nav__matrix-item{padding:1rem 0;width:100%}}.global-nav .global-nav__matrix-image{display:inline-block;height:auto;margin-bottom:1rem;margin-right:.75rem;max-height:2rem;max-width:2rem;position:relative;top:.1rem}@media (max-width:875px){.global-nav .global-nav__matrix-image{display:none}}.global-nav .global-nav__matrix-content{display:flex;flex:1 1 auto;flex-direction:column;margin:0;padding-right:1rem;width:100%}.global-nav .global-nav__matrix-title{line-height:2rem}@media (max-width:875px){.global-nav .global-nav__matrix-title{margin-bottom:0}}.global-nav .global-nav__matrix-desc{margin:-1rem 0 .1rem 2.75rem}@media (max-width:875px){.global-nav .global-nav__matrix-desc{display:none}}.global-nav .global-nav__link{color:#f7f7f7;display:inline-flex;text-decoration:none}.global-nav .global-nav__link:hover{text-decoration:underline}.global-nav .global-nav__split-list{-webkit-column-gap:2rem;column-gap:2rem;-webkit-columns:2;column-count:2;list-style:none;margin:0 0 1.5rem;padding:0}.global-nav .global-nav__split-list .global-nav__list-item{display:inline-block;margin:0;padding:.25rem 0;width:100%}.global-nav .global-nav__inline-list{margin:.5rem 0 0;padding:0 0 1rem 2.75rem}@media (max-width:875px){.global-nav .global-nav__inline-list{margin-top:0;padding:0}}.global-nav .global-nav__inline-list .global-nav__list-item{display:inline;font-weight:400;list-style:none;margin-right:1.25rem}.global-nav .global-nav__muted-heading{color:#f7f7f7;font-weight:400}@media (max-width:875px){.global-nav .global-nav__muted-heading{font-size:.8175rem}}.global-nav .global-nav__overlay{transition-duration:.5s;transition-property:opacity;transition-timing-function:cubic-bezier(.215,.61,.355,1);background-color:rgba(17,17,17,.4);height:100%;left:0;margin:0;opacity:0;pointer-events:none;position:fixed;width:100%;z-index:97}@media (max-width:875px){.global-nav .global-nav__overlay{display:none}}.global-nav .global-nav__overlay.show-overlay{opacity:1;pointer-events:all}@media (max-width:875px){body{padding-bottom:3rem;position:relative}.u-hide--mobile{display:none}}')}]);
```

Comment:
`global-nav.js` shouldn't be committed to the codebase. It should be copied in place from `node_modules` in a build step, and the copied file should be ignored with `.gitignore` (e.g. see [www.canonical.com](https://github.com/canonical-websites/www.canonical.com/blob/master/package.json#L7) or [microk8s.io](https://github.com/canonical-websites/microk8s.io/blob/master/package.json#L7))